docs/snippets/quickstart-install.md

```diff
@@ -38,22 +38,35 @@
 
 The `quickstart` plugin completes the following functions:
 
-1. **Checks if you have [`kind`](https://kind.sigs.k8s.io/docs/user/quick-start){target=_blank} installed,** and creates a cluster called `knative`.
-1. **Installs Knative Serving with Kourier** as the default networking layer, and nip.io as the DNS.
-1. **Installs Knative Eventing** and creates an in-memory Broker and Channel implementation.
+1. **Checks if you have the selected kubernetes instance installed,** and creates a cluster called `knative`.
+2. **Installs Knative Serving with Kourier** as the default networking layer, and nip.io as the DNS.
+3. **Installs Knative Eventing** and creates an in-memory Broker and Channel implementation.
 
-!!! todo "Install Knative and Kubernetes on a local Docker Daemon using `kn quickstart`"
+
+!!! todo "Install Knative and Kubernetes locally"
+=== "Using kind"
```

Comment:
Would you be able to indent lines 47 to 69 by 4 spaces. That way the section will be inside the green note the same as the section above it.

<img width="620" alt="Screen Shot 2021-11-17 at 13 42 35" src="https://user-images.githubusercontent.com/33425661/142212106-1b023a0e-76fd-4314-a6c5-294e41ce6e23.png">